benchmarks/http4k/src/main/kotlin/org/renaissance/http4k/Http4kBenchmark.kt

```diff
@@ -0,0 +1,109 @@
+package org.renaissance.http4k
+
+import kotlinx.coroutines.runBlocking
+import org.http4k.client.OkHttp
+import org.renaissance.Benchmark
+import org.renaissance.Benchmark.*
+import org.renaissance.BenchmarkContext
+import org.renaissance.BenchmarkResult
+import org.renaissance.BenchmarkResult.Validators
+import org.renaissance.License
+import org.renaissance.http4k.workload.WorkloadClient
+import org.renaissance.http4k.workload.WorkloadConfiguration
+import org.renaissance.http4k.workload.WorkloadServer
+
+@Name("http4k")
+@Group("kotlin")
+@Summary("Runs the http4k server and tests the throughput of the server by sending requests to the server.")
+@SupportsJvm("21")
+@Licenses(License.APACHE2)
+@Repetitions(20)
+@Parameter(
+    name = "host",
+    defaultValue = "localhost",
+    summary = "Host of the server."
+)
+@Parameter(
+    name = "port",
+    defaultValue = "8000",
+    summary = "Port of the server."
+)
+@Parameter(
+    name = "read_workload_repeat_count",
+    defaultValue = "5",
+    summary = "Number of read requests to generate."
+)
+@Parameter(
+    name = "write_workload_repeat_count",
+    defaultValue = "5",
+    summary = "Number of write requests to generate."
+)
+@Parameter(
+    name = "ddos_workload_repeat_count",
+    defaultValue = "5",
+    summary = "Number of ddos requests to generate."
+)
+@Parameter(
+    name = "mixed_workload_repeat_count",
+    defaultValue = "5",
+    summary = "Number of mixed requests to generate."
+)
+@Parameter(
+    name = "workload_count",
+    defaultValue = "100",
```

Comment:
For me, the value of around 450 caused the that benchmark runs about the same time as `finagle-http` which seems to me as a closest benchmark (the default value of 200 lead to runs with runtimes around 700ms which seems to me to be to quick).

Reply:
Good point, changed. :)